src/olympia/addons/models.py

```diff
@@ -607,9 +607,18 @@ def from_upload(cls, upload, platforms, channel=amo.RELEASE_CHANNEL_LISTED,
         addon = cls.initialize_addon_from_upload(
             parsed_data, upload, channel, user)
 
+        reviewer_flags_defaults = {}
         if upload.validation_timeout:
+            reviewer_flags_defaults['needs_admin_code_review'] = True
+        if (parsed_data.get('is_mozilla_signed_extension') and
+                addon.type != amo.ADDON_LPAPP):
```

Comment:
I think it's fine to pull this out into something like

```python
is_mozilla_signed = parsed_data.get('is_mozilla_signed_extension')

if is_mozilla_signed and addon.type != amo.ADDON_LPAPP:
    reviewer...
```

…to make it easier to read.